js/index.js

```diff
@@ -1 +1,163 @@
 // Your code goes here
+//click Event:******************************************************************************
+const allSections = document.querySelectorAll('section');
+
+allSections.forEach((banana) => {
+    return banana.addEventListener('click', function(event) {
+        event.stopPropagation();
```

Comment:
Good use of stopPropagation  to make the click event to affect the other sections.